src/backend/marsha/core/views.py

```diff
@@ -440,23 +442,65 @@ def post(self, request, *args, **kwargs):
             generated from applying the data to the template
 
         """
-        lti = LTI(self.request)
         try:
-            lti.verify()
-        except LTIException as error:
+            jwt_token = AccessToken(self.request.POST.get("jwt", ""))
```

Comment:
Most of the time a JWT token is used in the request `Authorization` header and [DRF](https://www.django-rest-framework.org/) provides mechanism to manage them. You have to look if it's possible to use them in a "classic" view.

Reply:
I'll try it, thanks